fab/src/main/res/drawable-v21/fab_cover_circle.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ripple android:color="#fff" xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:drawable="@drawable/fab_round_shape"/>
+</ripple>
```

Comment:
Please use 6- or 8-digit colour codes to keep consistency with the rest of the project.